include/rocksdb/statistics.h

```diff
@@ -394,7 +394,11 @@ enum Tickers : uint32_t {
   NON_LAST_LEVEL_READ_BYTES,
   NON_LAST_LEVEL_READ_COUNT,
 
+  // Number of block checksum verifications
   BLOCK_CHECKSUM_COMPUTE_COUNT,
+  // Number of corruptions detected by block checksum verifications
```

Comment:
I can change this to "Number of times RocksDB detected a corruption while verifying a block checksum"

Reply:
Or "Number of times RocksDB detected a corruption while verifying a block checksum. RocksDB does not remember corruptions so the same block corruption may be detected multiple times."

Reply:
Yes, that should be sufficient.